typescript/src/world-company-remuneration/basket-informations.ts

```diff
@@ -0,0 +1,46 @@
+const K = "K";
+
+/**
+ * Created by thomas on 02/12/2019.
+ * This class is a basket
+ */
+
+export class BasketInformations {
+    // The product of the basket
+    static map: Map<string, number> = new Map<string, number>()
+
+    addProductToBasket(product: string, price: number) {
```

Comment:
shoudl mention the return type

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway